pkgs/state-transition/src/transition.zig

```diff
@@ -158,25 +158,9 @@ fn process_attestations(allocator: Allocator, state: *types.BeamState, attestati
     var historical_block_hashes = std.ArrayList(types.Root).fromOwnedSlice(allocator, state.historical_block_hashes);
     var justified_slots = std.ArrayList(u8).fromOwnedSlice(allocator, state.justified_slots);
 
-    // prep the justifications map
-    var justifications = std.AutoHashMap(types.Root, []u8).init(allocator);
+    var justifications = try loadJustificationsMap(allocator, state, logger);
```

Comment:
it would be better strategy to declare justifications pointer here and pass it as this can likely grow to a big structure